core/cache/unit.go

```diff
@@ -74,7 +74,7 @@ func (u *Unit) Ports() []network.Port {
 // taking into account whether it is tracking a model branch.
 func (u *Unit) ConfigSettings() (charm.Settings, error) {
 	if u.details.CharmURL == "" {
-		return nil, errors.New("unit charm not set")
+		return nil, errors.New("unit charm must be set before retrieving config")
```

Comment:
Is 'unit charm' meaningful to people? It sounds a bit odd. vs "unit's charm URL" or something along those lines.

Reply:
Done as suggested.